mycroft/client/text/__main__.py

```diff
@@ -46,9 +46,21 @@ def main():
         log_dir = os.path.expanduser(config['log_dir'])
         start_log_monitor(os.path.join(log_dir, 'skills.log'))
         start_log_monitor(os.path.join(log_dir, 'voice.log'))
+        start_log_monitor(os.path.join(log_dir, 'audio.log'))
```

Comment:
Should this use a glob to detect all logs in the directory? Or possibly read log filenames from config?